sporkfish/searcher/move_ordering/composite_heuristic.py

```diff
@@ -0,0 +1,46 @@
+from enum import IntEnum
+from typing import List
+
+import chess
+
+from ...board.board import Board
+from .killer_move_heuristic import KillerMoveHeuristic
+from .mvv_lva_heuristic import MvvLvaHeuristic
+
+
+# TODO: To be tuned later
+# I'm personally not very big fan of letting thisbe configurable from config
+# People can input whatever nonsensical values
+# But perhaps it could be useful for testing when we first start out. TBD.
+class CompositeHeuristicScore(IntEnum):
+    KILLER_MOVE = 1
+    MVV_LVA = 2
+
+
+class CompositeHeuristic(MvvLvaHeuristic, KillerMoveHeuristic):
```

Comment:
Thinking about extendibility + sanity checking: 

In the future, this `CompositeHeuristic` class would inherit from ALL of the other "singular" move ordering classes, right?

Reply:
I am more and more convinced a [decorator pattern](https://refactoring.guru/design-patterns/decorator/python/example#lang-features) is what we want for our problem.

We can discuss further offline? I am keen to use this pattern - in fact quite keen to work on this PR, but as you know, I probs can only work on this say Weds onwards, don't want to block you.

A [youtube reference](https://www.youtube.com/watch?v=v6tpISNjHf8&t=188s) on decorator pattern, with example code in Java.

Reply:
Can you elaborate why? I don't see any pros/cons listed in your link.

Reply:
Discussed offline, raised #119 in case for future.

Reply:
Nicee - tyty. I am a bit ashamed to say I haven't put too much thoughts into it, my only argument is multiple inheritance doesn't seem very a nice pattern to extend. 

You are right, we have agreed on the design, and it looks great to me (most importantly, it works now). In favour to stick to your design unless we see significant issue.

Let's keep it [agile](https://en.wikipedia.org/wiki/Agile_software_development) :)

Reply:
Looks like we're already running into issues due to order of inheritance - python deities are agreeing with you. As discussed offline to refactor later